android/app/src/main/java/app/organicmaps/widget/recycler/MultilineLayoutManager.java

```diff
@@ -8,9 +8,12 @@
 
 public class MultilineLayoutManager extends RecyclerView.LayoutManager
 {
-  public MultilineLayoutManager()
+  private boolean reverseLayout = false;
+
+  public MultilineLayoutManager(int layoutDirection)
```

Comment:
Is there an alternative way to implement similar functionality using some system/androidx classes? CC @Jean-BaptisteC